.travis.yml

```diff
@@ -26,19 +26,28 @@ before_script:
   - |
       if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
       pip install git+https://github.com/SUPERAndroidAnalyzer/travis-cargo.git@fix-coveralls --user &&
-      export PATH=$HOME/.local/bin:$PATH
+      export PATH=$HOME/.local/bin:$PATH &&
+      (travis-cargo install rustfmt || true) &&
```

Comment:
The `|| true` would fail silently, which I think is not a good idea if we want builds to fail if *rustfmt* fails to install. I think it's better if we remove it.